library/ssl_tls13_generic.c

```diff
@@ -34,6 +34,10 @@
 #include "ssl_tls13_keys.h"
 #include "ssl_debug_helpers.h"
 
+#include "psa/crypto.h"
+#include "psa_crypto_hash.h"
```

Comment:
Note: if the call to `mbedtls_md_info_from_psa()` can be removed as I suspect, then this line can be removed too.